Makefile.toml

```diff
@@ -366,6 +366,25 @@ fi
 '''
 ]
 
+[tasks.check-lints]
+script = [
+'''
+rc=0
+
+# For rust first-party source code
+if ! cargo clippy \
+  --manifest-path ${BUILDSYS_TOOLS_DIR}/Cargo.toml \
```

Comment:
I guess the comment isn't exactly accurate yet. Are we planning to do this to first-party code in `/sources` as a follow up?

Reply:
Yes, I'd like first-party code to be covered as well, but it's more than I had the time to tackle today. I was working on an unrelated update to buildsys and got annoyed by the existing `clippy` warnings. 😀